projects/solar/inc/relay_fsm.h

```diff
@@ -12,12 +12,21 @@
 #include "solar_boards.h"
 #include "status.h"
 
+// The relay is disconnected  if current is
+// more than current thershold,
```

Comment:
It's actually disconnected if current is **less** than the current threshold right? Also typo: `thershold` should be `threshold`, and there's two spaces in the first comment line when there should be only one, and there's an extraneous comma at the end.

Reply:
done